public/app/plugins/datasource/cloudwatch/language_provider.ts

```diff
@@ -103,6 +103,18 @@ export class CloudWatchLanguageProvider extends LanguageProvider {
     ];
   }, 30 * 1000);
 
+  isStatsQuery(query: string): boolean {
+    const grammar = Prism.languages['cloudwatch'];
```

Comment:
Why don't we use `this.getSyntax()` here?

Reply:
good catch